### ISSUE TITLE
Weapons acting as if equipped when dropped

### ISSUE THREAD

**Issue by Caradog** [CombatExtendedRWMod/CombatExtended#1078]
Using CE:FT 1.0 (their issue tracker is closed with the merger) - After dropping a weapon its still behaving as if its equipped to the original pawn, having the UI elements to change its fire mode and reload/unload (unloading seems to either cause the ammo to drop at the pawns location or into their inventory).

Have created a Hugs log: https://gist.github.com/HugsLibRecordKeeper/12793422e71ea69b0a096aee5e9185c3

**Comment by Alicecomma** [comment]
Nothing of use in the error log, it's full of log spam from "ResearchTree", nothing from CombatExtended.

**Comment by Caradog** [comment]
Enabled verbose logging, cleared the log and tried again: https://git.io/JvaYr

Looks like what ever is causing it isn't getting logged? Definitely occurring though as I've repeated it a number of times while trying to get the log to flag something

**Comment by Alicecomma** [comment]
No, your log is still being spammed with "ResearchTree", meaning the log stops logging anything due to the 1000 message limit before anything that is of use is logged.

**Comment by Caradog** [comment]
Actually if you compare the two logs, they're identical to line 5269 where I restarted it logging manually, I then had a pawn repeatedly pick up and drop a weapon with verbose logging enabled

**Comment by Alicecomma** [comment]
The lines from 5269 similarly have no information in them, as they are related to the sound system switching on and off.
